Makefile

```diff
@@ -12,4 +12,8 @@ lint: fmt
 
 .PHONY: fmt
 fmt:
-	bash -c "diff -u <(echo -n) <(gofmt -d -s .)"
+	gofmt -w -s .
+
+.PHONY: generate
+generate:
+	openapi-generator generate -i openapi/dcos.yaml -g go -o dcos --skip-validate-spec -D packageName=dcos,withGoCodegenComment=true -t templates
```

Comment:
Not sure we want to `--skip-validate-spec` feels like that's quite important for us while we stabilize the spec.

Reply:
I agree, I added it to be able to get something generated, as there are some missing description in the specs